src/pytest_dbfixtures/factories/dynamodb.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2013 by Clearcode <http://clearcode.cc>
```

Comment:
So it's copied code published in 2013, with unoriginal changes only? Or should it be `2016`?